### ISSUE TITLE
Bootstrap array overlap form input

### ISSUE THREAD

**Issue by ericduhamel** [twbs/bootstrap#18374]
Hi,

I use bootstrap 3.
I designed the side with a left panel for the menu and the main content to the right.

Both section is under a container-fluid and row fluid.
My main part is under a tab-content and tab-pane.

When i use a table in the panel who have many line, it's overlap my form under it.
http://jsfiddle.net/z7n1b27z/

image
http://picpaste.com/bootstrap-table-AEouqdFx.jpeg

With only the pannel, that work fine. http://jsfiddle.net/33h53t4m/

Is there a incompatibility with theses components?

**Comment by twbs-lmvtfy** [comment]
Hi @ericduhamel!

You appear to have posted a live example (http://fiddle.jshell.net/m0pey3ev/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 73, column 25: [E013](https://github.com/twbs/bootlint/wiki/E013): Only columns (`.col-*-*`) may be children of `.row`s
* line 49, column 9: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s
* line 50, column 9: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s
* line 49, column 9: [W009](https://github.com/twbs/bootlint/wiki/W009): Using empty spacer columns isn't necessary with Bootstrap's grid. So instead of having an empty grid column with `class="col-sm-2"` , just add `class="col-sm-offset-2"` to the next grid column.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by ericduhamel** [comment]
fixed

**Comment by ericduhamel** [comment]
seem like bootstrap table don't  add padding-bottom to 40px...

**Comment by cvrebert** [comment]
"bootstrap-table" is a third-party project that is not maintained by us nor officially affiliated with us.
Please use their issue tracker instead: https://github.com/wenzhixin/bootstrap-table/issues

**Comment by cvrebert** [comment]
Superseded by https://github.com/wenzhixin/bootstrap-table/issues/1760
